docs/kaneai-smartui-visual-testing.md

```diff
@@ -0,0 +1,116 @@
+* **Screenshot Name (Required):** You must provide a descriptive name for the screenshot.
+* **Mis-Match Threshold (Optional):** Set an acceptable percentage of difference below which changes are auto-approved. The default is 0%.
+* **Screenshot Type (Optional):** Choose between **View Port** (default) or **Full Page**.
+
+Click **Create** to add the step to your test flow.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/smartui/configure-screenshot.png').default} alt="Configure Screenshot Settings" className="doc_img"/>
+
+## Analyzing Baselines and Regressions
+
+### Step 4: Establish the Baseline
+The first time the test is executed, it captures an initial screenshot that becomes the **Baseline** for all future comparisons. You can view this image by clicking **Show Baseline**.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/smartui/establish-baseline.jpeg').default} alt="Baseline screenshot created" className="doc_img"/>
+
+### Step 5: Rerun and Analyze the Comparison
+When you rerun the test, a new screenshot is captured and compared against the baseline. Click **Show Comparison** to open the SmartUI view. This interface highlights any visual differences found between the two images.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/smartui/analyze-comparison.png').default} alt="Analyze the visual comparison between baseline and new screenshot" className="doc_img"/>
+
+### Step 6: Approve or Reject Changes
+Within the comparison view, you can review the changes and decide on a verdict.
+* **Reject Screenshot:** If the changes are unintended defects, you may reject the comparison.
+* **Approve Screenshot:** If the changes are intentional, you may accept the comparison. This will update the baseline with the new image.
+
+You can also adjust project settings like **Compare Type** and **Mis-Match Threshold** directly from this view.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/smartui/approve-reject-changes.jpeg').default} alt="Approve or Reject Screenshot Changes" className="doc_img"/>
+
+## Saving and Automating Your Visual Test
+
+### Step 7: Save Your Test Case
+Once your test flow is complete, click **Finish test**. You will be prompted to save the session, where you can define the **Project**, **Folder**, **Test Name**, and **Description**.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/smartui/save-test-case.png').default} alt="Save the test case session" className="doc_img"/>
+
+### Step 8: Schedule the Test for Automation
+You can access your saved tests via the **Test Manager**. To automate execution, navigate to **Test Runs** and schedule the finalized test[cite: 225]. When creating the run, provide a name and select **KaneAI Generated** as the `Type`. You can then set a recurring schedule for execution. Scheduled runs will automatically capture screenshots and store them in the SmartUI project for continuous comparison.
```

Comment:
Reference existing doc https://www.lambdatest.com/support/docs/kaneai-hyperexecute-test-run-execution/ for Test Runs.

Reference existing doc for scheduled runs - https://www.lambdatest.com/support/docs/kaneai-scheduled-test-runs/

Remove [cite: 225] - Not sure what that is. @RushilK7